Description: Fix version for 1.1.54188 stable
Author: Jean Baptiste Favre <debian@jbfavre.org>
Origin: other
Last-Update: 2017-03-20
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/dbms/cmake/version.cmake
+++ b/dbms/cmake/version.cmake
@@ -1,6 +1,6 @@
 #This strings autochanged from release_lib.sh :
-set(VERSION_DESCRIBE v1.1.54189-testing)
-set(VERSION_REVISION 54189)
+set(VERSION_DESCRIBE v1.1.54190-stable)
+set(VERSION_REVISION 54190)
 #===end of autochange
 
 set (VERSION_MAJOR 1)
